content/en/docs/components/misc/nuclio.md

```diff
@@ -190,7 +190,7 @@ def nuc_pipeline(
 ```
 the code above assumes nuclio was deployed into the `nuclio` namespace on the same cluster, when using a remote cluster or a different namespace you just need to change the `dashboard` URL.
```

Comment:
Line 190: please take a look at this also

- Capitalize the first letter in the first word of the sentence (`The code...`)
- Split the complex sentence into two simpler ones for readability.


```suggestion
The code above assumes nuclio was deployed into the `nuclio` namespace on the same cluster. When using a remote cluster or a different namespace you just need to change the `dashboard` URL.
```

Reply:
Done :heavy_check_mark: